LICENSE-APACHE

```diff
@@ -186,7 +186,8 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright [yyyy] [name of copyright owner]
+Copyright 2015 rust-mersenne-twister developers
```

Comment:
I filled in the same copyright from the `LICENSE-MIT` file here.
Given the APACHE-2.0 license blurb with copyright in all the source files, I believe this change was intended.